gen/cmd/Cargo.toml

```diff
@@ -29,7 +29,7 @@ static = [ "autocxx-engine/static" ]
 
 [dependencies]
 autocxx-engine = { version="=0.16.0", path="../../engine" }
-clap = "~2.33"
+clap = { version = "3.1.2", features = ["cargo", "derive"] }
```

Comment:
Why do we need these `cargo` and `derive` features? It's desirable to minimize the number of transitive dependencies (especially because some autocxx users use non-cargo build systems) and I suspect these bring in some additional dependencies that we might not need?

Reply:
(I figured out the reason for `cargo` but still not sure about `derive`).